### PR TITLE
fix: incorrect input names

### PR DESCRIPTION
**Pull request by sumitarora**
Fixes: https://github.com/rangle/angular-devtools/issues/793

To test add below inputs to `demo-app-component.ts`

```ts
  @Input('a') _a = 'input a';
  @Input('b') _b = 'input b';
  @Input() _d = 'input d';

  a = 'prop a';
  c = 'prop c';
```

image

### CODE REVIEW

**Comment by mgechev** [comment]
@sumitarora would you add tests for this fix? Unit and e2e ideally. Looks like we have some failures in our e2e job.

**Comment by sumitarora** [comment]
@mgechev added

File: cypress/integration/view-component-metadata.e2e.js [hunk 1]
```diff
@@ -36,5 +36,71 @@ describe('Viewing component metadata', () => {
         .last()
         .should('have.text', ' Change Detection Strategy: Default');
     });
+
+    it('should display correct set of inputs', () => {
```
Comment (mgechev) [inline cypress/integration/view-component-metadata.e2e.js:40]: Can we also add tests for updating the properties?

Comment (sumitarora) [reply cypress/integration/view-component-metadata.e2e.js:40]: Those are already in property-update and property-edit tests. https://github.com/rangle/angular-devtools/tree/main/cypress/integration

Comment (mgechev) [reply cypress/integration/view-component-metadata.e2e.js:40]: I was thinking of updating renamed inputs to verify both reads and writes work.

**Review by mgechev — APPROVED** [review]

**Comment by mgechev** [comment]
Will release once I get back from 🌴

**Pull request merged**